### PR TITLE
Added: FAQ Ukrainehilfe

### PR DESCRIPTION
**Pull request by GoliathLabs**
Added a post with frequently asked questions as previously discussed in chat

### CODE REVIEW

**Comment by awlx** [comment]
Should we add some Links to facebook, Twitter etc posts and the map to show what we can do in a short time?

**Comment by T0biii** [comment]
i added some anchors (can be renamed) so you can link(z.B. #Grundsaetzliches) directly to the anchor
also made the headings bigger and renamed SimKarte

adding some links to facebook and twitter etc. sounds got to me.

File: _posts/2022-03-25-internet-fuer-fluechtlinge.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Das kommt auf den Umfang der Installation an und ist schwer abschätzbar im Vorfeld. Wenn ihr einen günstigen Freifunk-Router selbst konfigurieren und aufstellen könnt, entstehen keine weiteren Kosten. Professionelle Hardware für größere Einrichtungen ist teuer; hier muss im Einzelfall über die Finanzierung gesprochen werden.
+
+
+### Wie sieht es mit der Haftung aus? {#Haftung}
+
+Freie Netze München e.V. tritt als Provider auf und übernimmt damit auch die Haftungsfrage. Mehr zu den rechtlichen Aspekten und Sicherheit im Freifunk ist im FAQ zu lesen: <https://ffmuc.net/wiki/doku.php?id=knb:faq#rechtliches>
+Grundsätzlich gilt: Wir verhindern oder erschweren keine legalen Maßnahmen der Strafverfolgung. Falls rechtlich begründet, arbeitet Freifunk München vollumfänglich mit Justiz und Polizei zusammen.
+
+
+### Wie wird das finanziert? {#Finazierung}
```
Comment (fadenb) [inline _posts/2022-03-25-internet-fuer-fluechtlinge.markdown:76]: Typo, ein `n` fehlt in Finazierung

**Review by fadenb — APPROVED** [review]

**Review by fadenb — APPROVED** [review]

**Comment by GoliathLabs** [comment]
> Should we add some Links to facebook, Twitter etc posts and the map to show what we can do in a short time?

@awlx which posts would you suggest?

**Review by awlx — APPROVED** [review]

**Pull request merged**